### ISSUE TITLE
[Feature Request] Support to optionally run computationally expensive tasks externally (e.g. HPC, GPU clusters...)

### ISSUE THREAD

**Issue by pedro-cmat** [vantage6/vantage6#431]
**Problem description**
The vantage6 node runs the algorithms directly on the same machine where it's running. However, some algorithms may have computational requirements that aren't fulfilled by this machine and need the additional power of an HPC or GPU cluster. In addition, configuring the vantage6 node on these machines may be unsuitable or unattainable.

**Desired solution**
Being able to send the tasks that need additional computer power to an external machine available at the organization. This should account for the differences in infrastructure that can be found at each organization.

**Additional context**
We've worked on a solution for a project that may give some ideas/suggestions (WIP: https://github.com/pedro-cmat/vantage6-node):
- Each node will have a new docker image (a wrapper) to submit the tasks to their external machine (e.g. image to submit the tasks to a GPU cluster using open shift)
- The algorithm image will continue to be the same for all nodes.
- Each vantage6 node has a new parameter in the configuration file to map a placeholder to the new docker image.
- When requesting a new task ('post_task'), the user should use the placeholder in the field 'image'.
- The vantage6 node will spin up the wrapper docker image that will send the request to the external machine (to run the algorithm) and later retrieve the result.
- The container for the wrapper would need access to the external machine - e.g. whitelisting

Happy to hear your comments and discuss this further.

**Comment by frankcorneliusmartin** [comment]
Related: #162

**Comment by frankcorneliusmartin** [comment]
Probably dependent on: #248

**Comment by bartvanb** [comment]
Superseded #1679
